citations/citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -912,6 +930,13 @@ protected void doPostJsonResponse(ParameterParser params,
 
 	}
 
+	private void logErrorMessage(Map<String, Object> jsonMap, CitationCollection collection, String errorMessage) {
+		Map<String, Object> result = new HashMap<>();
+		result.put("Error: ", new String[]{new Date() + ": Invalid nested list for collection with id: " + collection.getId()});
+		jsonMap.putAll(result);
+		logger.error(errorMessage);
```

Comment:
It's not necessarily a citation that's causing problems, it could be a updating of a section. There's plenty of context in the errorMessage like collection id, the type of section etc.